libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt

```diff
@@ -125,15 +127,16 @@ internal object MapboxNavigationTelemetry {
     private val feedbackEventCacheMap = LinkedHashMap<String, NavigationFeedbackEvent>()
 
     private var needHandleReroute = false
-    private var sessionState: NavigationSession.State = IDLE
+    private var sessionState: NavigationSessionState = Idle
     private var routeProgress: RouteProgress? = null
     private var originalRoute: DirectionsRoute? = null
     private var needStartSession = false
+    private var eventsSessionId: String = ""
```

Comment:
This is never set as far as I can tell. Shouldn't we assign it when state changes?